sensor_msgs/Image image_raw
---
image_yolo image_detect